tests/span_tests.cpp

```diff
@@ -158,27 +158,27 @@ TEST(span_test, from_pointer_length_constructor)
     int arr[4] = {1, 2, 3, 4};
 
     {
-        for (int i = 0; i < 4; ++i)
+        for (std::size_t i = 0; i < 4; ++i)
```

Comment:
Technically this one should still work as int because the loop ensures it is in the bounds [0, 4). Is changing to size_t for warning resistance?

Reply:
(Occurs elsewhere in these tests)

Reply:
Yeah, as an int this was generating an additional warning. Just trying to avoid that.